### PR TITLE
Fixed a javascript error in the mobile viewer example that prevented …

### PR DESCRIPTION
**Pull request by mmcev106**
…an alert from being displayed

### CODE REVIEW

**Review by glittle2638 — APPROVED** [review]

File: examples/mobile-viewer/viewer.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-if (!PDFJS.PDFViewer || !PDFJS.getDocument) {
+if (typeof PDFJS == 'undefined' || !PDFJS.PDFViewer || !PDFJS.getDocument) {
```
Comment (timvandermeij) [inline examples/mobile-viewer/viewer.js]: Use `===` (strict equality) here like we do elsewhere.

**Review by timvandermeij — COMMENTED** [review]
Looks good with the comment fixed.

File: examples/mobile-viewer/viewer.js (hunk 1 above)
Comment (mmcev106) [reply examples/mobile-viewer/viewer.js]: Done.  Anything else?

**Review by timvandermeij — APPROVED** [review]
Good to merge after you've squashed the two commits into one. Refer to https://github.com/mozilla/pdf.js/wiki/Squashing-Commits if you're not familiar with the process. Thank you!

**Comment by mmcev106** [comment]
I believe I've squashed them appropriately.  Let me know if you'd like me to make any other changes.

**Review by Snuffleupagus — APPROVED** [review]
Thank you!

**Pull request merged**